lametro/models.py

```diff
@@ -722,6 +722,17 @@ def ecomment_message(self):
             return self.UPCOMING_ECOMMENT_MESSAGE
 
 
+    @property
+    def accepts_live_comment(self):
+        meetings_without_live_comment = (
```

Comment:
not a requirement but making this a set with

{
           'Measure R Independent Taxpayer Oversight Committee',
            'Measure M Independent Taxpayer Oversight Committee',
            'Independent Citizen’s Advisory and Oversight Committee',
}

make intent slightly clearer.

Reply:
Done in https://github.com/datamade/la-metro-councilmatic/pull/823/commits/8c8383b851a5df352d9c5d361edcc8a5168c9979.